node-backend/app/controllers/v1/CoreValueController.js

```diff
@@ -33,6 +40,7 @@ module.exports.create = (req, res) => {
         });
     })
     .catch((err) => {
+      logger.error(err);
```

Comment:
Please follow below structure for error logging
**********************************
Error in inserting core value
user id: <user_id>
core value data: <data>
error : <error details>
********************************